lib/middleware/errors.js

```diff
@@ -12,31 +13,27 @@ function canWithstand(error) {
   }
 }
 
-function _renderError(err, req, res, next, noStack) {
-  res.status(err.status || 500);
+function _applicationError(err, req, res, next) {
+  err.status = err.status || 500;
+
   logger.log('error', {'Application error': err.message});
   logger.log('error', {'Error stack': err.stack});
 
-  const errorPath = config.KAT_ERROR_PATH;
-  res.redirect(`${errorPath}/${res.statusCode}`);
+  next(err);
```

Comment:
:+1: 
Thanks for explaining. I just suggested that because that's what I did to test it locally (and no other change was needed on the FE app)